<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Step9</class>
 <widget class="QWidget" name="Step9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1445</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>725</x>
     <y>0</y>
     <width>720</width>
     <height>480</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>376</y>
      <width>101</width>
      <height>47</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>step9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>376</y>
      <width>121</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string> 竞争性矩阵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>376</y>
      <width>81</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string>期望位次</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>372</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关键度</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Step1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Step9</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>485</y>
     <width>1445</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
